src/main/java/RacingGame.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class RacingGame<puiblic> {
+
+    private int[] carPositions;
+    private int tryNumber;
+
+    public void setTryNumber(int tryNumber) {
+        this.tryNumber = tryNumber;
+    }
+
+    public void setCarNumber(int carNumber) {
+        carPositions = new int[carNumber];
+    }
+
+    private void inputCarNumber() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("자동차 대수는 몇대인가요?");
+        int carNumber = scanner.nextInt();
+        setCarNumber(carNumber);
+    }
+
+    private void inputTryNumber() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("시도할 회수는 몇대인가요?");
+        int tryNumber = scanner.nextInt();
+        setTryNumber(tryNumber);
+    }
+
+    public int getRandomNumber() {
+        Random random = new Random();
+        return random.nextInt(10);
+    }
+
+    public int goOrStop(int randomNumber) {
```

Comment:
```suggestion
    public boolean isMove(int randomNumber) {}
```

제가 보기엔 boolean 함수같습니다. boolean 함수 컨벤션은 is~입니다. 

true false를 리턴 받는 함수를 편의에 따라서 1, 0 으로 변환하는 작업은 지양해주세요